lib/QubitWebResponse.class.php

```diff
@@ -23,9 +23,9 @@ class QubitWebResponse extends sfWebResponse
       * Sets title for the current web response.
       *
       * @param string  $title   Title name
-      * @param bool    $escape  true, for escaping the title
+      * @param bool    $escape  false, for escaping the title
       */
-     public function setTitle($title, $escape = true)
+     public function setTitle($title, $escape = false)
```

Comment:
Nitpick: I think the indenting is wonky, it's using 5 and 9 space indents in my editor. 😕